jac-uk/admin
# #1709 admin main page improvements

## What's included?
Closes #1709 

Design: https://www.figma.com/file/pR7HqskdIISgNGEw2SAZJC/1709-Admin-UI-Main-Page-Improvements?node-id=0-1&t=RYDrzbAY1mLelbD4-0

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Leader team:
1. Find an exercise that is ready to submit for approval.
2. Submit for approval.
3. Approve or reject the exercise:
    - If the exercise is approved, check if you can publish it on the website.
    - If the exercise is rejected, check if you can give reasons.

Normal user:
1. Find an exercise that is ready to submit for approval.
2. Submit for approval.
3. Wait for the exercise to be approved:
    - If the exercise is rejected, check if you can see the reasons and be able to submit for approval again.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:
- Leader team:

    https://user-images.githubusercontent.com/79906532/226648502-e8e57de1-88a7-4a5a-a5f6-921aec1bf0fa.mov

- Normal user:

    https://user-images.githubusercontent.com/79906532/226650035-7a5f56e7-829e-4802-b72d-52d766db1ef3.mov

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
